charts/pega/README.md

```diff
@@ -659,6 +668,7 @@ installer:
   upgrade:
     upgradeType: "out-of-place"
     targetRulesSchema: "rules_upgrade"
+    targetDataSchema: "temp_data_schema" (leave blank for releases earlier than 8.4)
```

Comment:
can we remove this as we haven't merged `tempDataSchema` support changes to master yet.

Reply:
yes...made changes to honor this comment